### ISSUE TITLE
cargo fmt: missing linebreak in error message

### ISSUE THREAD

**Issue by matthiaskrgr** [rust-lang/rustfmt#3644]
`./target/debug/cargo-fmt -p wasd`

The error message is:
````
[...]
ARGS:
    <rustfmt_options>...    Options passed to rustfmt%
````
looks like there is a final linebreak missing after the --help output.
(it should be `Options passed to rustfmt\n` instead of `Options passed to rustfmt`)

![rustfmt](https://user-images.githubusercontent.com/476013/59969405-cd8af580-954c-11e9-8572-178f17ce3575.png)

**Comment by nootanghimire** [comment]
Heya! 

Really new to rust, but would like to start contributing small things! I looked into the code, and it seems  this could either be solved by adding:

```rust
app.after_help("\n");
```

to  `cargo-fmt/main.rs:98` -- or maybe solving it for everyone in the `clap` crate.

It would be great if anyone could comment on which approach would be better.

Cheers!

**Comment by nootanghimire** [comment]
Turns out that the `after_help` hook adds a newline at the end without having to explicitly writing it, I'll create a PR.  :)

**Comment by topecongiro** [comment]
Closed via #3650. @nootanghimire Thank you for your work!
